### DISCUSSION TITLE
policy 的翻譯

### DISCUSSION THREAD

**Discussion by robinsonweng** [python/python-docs-zh-tw#826] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

policy

### 術語翻譯

政策, 方針, 策略

### 說明

在某些 context 下, 翻譯為 `政策` 有點不合適, 例如 [asyncio](https://docs.python.org/3/library/asyncio.html) 以及 [asyncio-policy](https://docs.python.org/3/library/asyncio-policy.html)

### 參考資料

- [微軟將 policies 翻譯為 '原則'](https://learn.microsoft.com/zh-tw/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.4)

- [在 python 的 venv libary 文件當中, 則是翻譯為策略](https://docs.python.org/zh-tw/3/library/venv.html)
    > 有關更多資訊，請參閱[關於執行策略](https://go.microsoft.com/fwlink/?LinkID=135170)。

**Comment by ken71301** [comment]
目前字典的定義為：a set of ideas or a plan of what to do in particular situations that has been agreed to officially by a group of people, a business organization, a government, or a political party.
因此，若使用語境為“使用別的公司的policy”，確實可以翻譯為政策
但一般來說，以 [policies頁面](https://docs.python.org/3/library/asyncio-policy.html)為例，此處的語意應更接近於`原則`或`方針`
3/30討論結果：翻譯為`原則`
